src/Selenium2Library/locators/__init__.py

```diff
@@ -1,12 +0,0 @@
-from .elementfinder import ElementFinder
-from .customlocator import CustomLocator
-from .tableelementfinder import TableElementFinder
-from .windowmanager import WindowManager
```

Comment:
In my opinion exposing different finders and locators directly on the `locators` package level was a good idea. In general I prefer packages exposing their public API in the `__init__` file. After this change imports like

```python
from Selenium2Library.locators import WindowManager
```

need to be changed to

```python
from Selenium2Library.locators.windowmanager import WindowManager
```

and that is definitely not an enhancement.